tests/lib/helpers.js

```diff
@@ -2,20 +2,72 @@
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
 const webpack = require("webpack");
-// import webpack from "webpack";
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const version = require("../../package.json").version;
 
 /*
   Returns a promise that resolves once webpack has compiled all the addon
   configs have been built.
 */
-// export default function
 
 function compileWebpack() {
   return new Promise((resolve, reject) => {
-    const configs = ["chrome", "firefox", "firefox-mobile", "opera"].map(item =>
-      require(`../../${item}/webpack.config.js`)
+    const config = ["chrome", "firefox", "firefox-mobile", "opera"].map(
+      platform => {
```

Comment:
Hmm, I wonder if there's some way we can essentially invoke `webpack(config)` with each version as `env`? That way we don't have to repeat what is essentially `webpack.config.js` here inline (which is an opportunity for us to get out of sync and introduce accidental bugs).

Reply:
Yeah, I don't love this. I stared at it for a while, but I couldn't quite get there on this pass. The config/configs constant has to be an object/array of objects for the API, but using env in the config file means that has to export a function to webpack rather than an object. Plus the way the env variable gets passed is ... weird to me. I'm not sure yet where I could find it in this approach. 

It may be that figuring out how to work up a merge configuration with two files (one that fills in the template, maybe, and one that pulls the template into an empty common config?) would make this less ick.